boards/1bitsy_stm32f415rgt.json

```diff
@@ -0,0 +1,39 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F4 -DSTM32F415xx -DSTM32F40_41xxx",
+    "f_cpu": "168000000L",
+    "ldscript": "stm32f405x6.ld",
+    "mcu": "stm32f415rgt",
+    "variant": "stm32f415xx"
+  },
+  "frameworks": [
+    "mbed",
```

Comment:
This manifest is not compatible with `mbed`. Please remove this framework.

Reply:
Sounds good. Removed. :)